coal/mineral.py

```diff
@@ -118,16 +138,20 @@ def classifyImage(imageFilename, classifier, classifiedImageFilename):
     # save the classified image to a file
     spectral.io.envi.save_classification(classifiedImageFilename, classifiedImage, ???)
 
-    # TODO close files ?
-
 def classifyImages(imageFilenames, classifier, classifiedImageFilenames):
-
     """
     Classify a set of AVIRIS images using a trained classifier and save the results to files.
+    Args:
+       imageFilenames (str): the images to be classified
+       classifier (PerceptronClassifier): the classifier
+       classifiedImageFilenames (str): the files which to save the classified images
```

Comment:
i expected that `imageFilenames` and `classifiedImageFilenames` would be a *list of strings*. for example, `["foo.hdr","bar.hdr","baz.hdr"]` and `["foo-minerals.hdr","bar-minerals.hdr","baz-minerals.hdr"]`. instead of a second list it would be possible to pass a renamer function, but i figured it would be simpler if the caller handled naming.